src/Foundation/Enum.cs

```diff
@@ -1380,6 +1385,9 @@ public enum NSFileProtectionType {
 		CompleteUnlessOpen,
 		[Field ("NSFileProtectionCompleteUntilFirstUserAuthentication")]
 		CompleteUntilFirstUserAuthentication,
+		[Watch (10, 0), TV (17, 0), NoMac, iOS (17, 0), MacCatalyst (17, 0)]
+		[Field ("NSFileProtectionCompleteWhenUserInactive")]
+		CompleteWhenUserInactive,
 		[Field ("NSFileProtectionNone")]
 		None,
```

Comment:
There is a breaking change

![image](https://github.com/xamarin/xamarin-macios/assets/204671/2fd42245-7fd2-4ec9-a8e0-198c0bdb56e4)


```suggestion
		[Field ("NSFileProtectionNone")]
		None,
		[Watch (10, 0), TV (17, 0), NoMac, iOS (17, 0), MacCatalyst (17, 0)]
		[Field ("NSFileProtectionCompleteWhenUserInactive")]
		CompleteWhenUserInactive,
```